Source/SwiftLintFramework/Rules/VariableNameMinLengthRule.swift

```diff
@@ -1,27 +1,40 @@
 //
-//  VariableNameMinLengthRule.swift
+//  VariableNameMinLengthNewRule.swift
```

Comment:
Yeah, this was done accidentally as I originally created a new rule `VariableNameMinLengthNewRule` to ensure the results of both rules were the same. Then I consolidated by deleting the old rule. I fixed and reattributed.